alibuild_helpers/build.py

```diff
@@ -174,11 +185,7 @@ def syncToLocal(self, p, spec):
       self.doneOrFailed.append(spec["hash"])
       return
 
-    cmd = format("mkdir -p %(hd)s && "
-                 "mkdir -p %(ld)s",
-                 hd=spec["tarballHashDir"],
-                 ld=spec["tarballLinkDir"])
-    execute(cmd)
+    execute(" ".join(("mkdir", "-p", spec["tarballHashDir"], spec["tarballLinkDir"])))
```

Comment:
why not simply `format("mkdir -p %s %s" % (spec["tarballHashDir"], spec["tarballLinkDir"]))`?

Reply:
OK, I've changed it!